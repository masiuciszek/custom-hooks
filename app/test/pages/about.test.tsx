import React from "react";

import { render } from "@testing-library/react";
import Layout from "../../components/layout";
import AboutPage from "../../pages/about";
import SiteProvider from "../../context/site.context/Site.context";

describe("Home Page", () => {
  test("should match Snapshot ", () => {
    const about = render(
      <SiteProvider>
        <Layout>
          <AboutPage />
        </Layout>
      </SiteProvider>
    );
    expect(about.asFragment()).toMatchSnapshot();
  });
});
