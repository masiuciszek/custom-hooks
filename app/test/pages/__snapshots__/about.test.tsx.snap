// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should match Snapshot  1`] = `
<DocumentFragment>
  <nav
    class="navbar-sc-8gcs5t-0 dokJbT main-navbar"
  >
    <div
      class="Texts__SecondaryTitle-sc-5d7d2m-0 elCLXM"
    >
      <h4>
        React Hooks
      </h4>
    </div>
    <ul
      class="NavStyles__NavListStyles-q0dme3-0 kVznwr"
    />
    <ul
      class="NavStyles__NavListStyles-q0dme3-0 NavStyles__SmallListStyle-q0dme3-2 bkjBKr"
      style="transform: translateY(-1000%);"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          About 
        </a>
      </li>
      <li>
        <a
          href="/faq"
        >
          Faq
        </a>
      </li>
    </ul>
    <span
      id="nav-icon"
    >
      üçî
    </span>
  </nav>
  <div
    class="layout__Page-sc-1lvij4u-0 kHeHsY"
  >
    <main
      class="layout__Main-sc-1lvij4u-1 hAlvDb"
    >
      <section
        class="title-sc-2igzia-0 cFwbxr About-Title"
      >
        <h1>
          React Hooks
        </h1>
        <h3>
          <span>
            React
          </span>
           
          <span>
            hooks
          </span>
           
          <span>
            with
          </span>
           
          <span>
            Marcell
          </span>
        </h3>
      </section>
      <div>
        <h1
          data-testid="h1"
        >
          0
        </h1>
        <button
          data-testid="add"
        >
          add
        </button>
        <button
          data-testid="dec"
        >
          decq
        </button>
        <button
          data-testid="reset"
        >
          reset
        </button>
      </div>
    </main>
  </div>
  <footer
    class="footer-p4wu29-0 OtdOM MainFooter"
  >
    <div
      class="Texts__SecondaryTitle-sc-5d7d2m-0 elCLXM"
    >
      <h4>
        React Hooks
      </h4>
    </div>
    <div
      class="Footerstyles__FooterListWrapper-sc-1sn5qiw-0 hsnCZF"
    />
  </footer>
</DocumentFragment>
`;
